### ISSUE TITLE
Release version for production env

### ISSUE THREAD

**Issue by kedar-githit** [microsoftgraph/aspnetcore-connect-sample#61]
Hey, this sample you created is fantastic. I am developing something similar to this. I tried your sample app too. Can you please suggest me what alteration I must do to deploy this sample on production environment?
It would be really helpful.
Thankyou in advance.

**Comment by mark-szabo** [comment]
If you just want to deploy the sample in a prod env, you don't need to do anything. You can publish into an Azure App Service from VS for example.
If you want to integrate Microsoft Graph into your solution, you'll need to do something similar to this sample, but the code here should be prod ready. 
Please keep in mind, that we are upgrading this sample right now, you should be able to see improvements in a few weeks.

**Comment by kedar-githit** [comment]
Actually I am developing a single sign on functionality with AzureAD, I was
able to create it successfully but later on I encountered 'The Owin bug' in
the authentication lib used and MS themselves suggested to avoid using ADAL
library(which I have used in my app) So I was looking with the MSAL
approach and found your sample. And now I am trying to deploy it on
production server and if it works out fine I'm looking forward to mimic
those functionalities in my app.
Thankyou for your earliest response.
